<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="actionBarLayout"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabMain">
        <attribute name="title">
         <string>&amp;Main</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_Main">
         <item>
          <widget class="QLabel" name="transactionFeeInfoLabel">
           <property name="text">
            <string>Optional transaction fee per kB that helps make sure your transactions are processed quickly. Most transactions are 1 kB. Fee 0.01 recommended.</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutFee">
           <item>
            <widget class="QLabel" name="transactionFeeLabel">
             <property name="text">
              <string>Pay transaction &amp;fee</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="buddy">
              <cstring>transactionFee</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="BitcoinAmountField" name="transactionFee"/>
           </item>
           <item>
            <spacer name="horizontalSpacerFee">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="reserveBalanceInfoLabel">
           <property name="text">
            <string>Reserved amount does not participate in staking and is therefore spendable at any time.</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutReserveBalance">
           <item>
            <widget class="QLabel" name="reserveBalanceLabel">
             <property name="text">
              <string>Reserve</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="buddy">
              <cstring>reserveBalance</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="BitcoinAmountField" name="reserveBalance"/>
           </item>
           <item>
            <spacer name="horizontalSpacerReserveBalance">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="bitcoinAtStartup">
           <property name="toolTip">
            <string>Automatically start ArtBoomerang after logging in to the system.</string>
           </property>
           <property name="text">
            <string>&amp;Start ArtBoomerang on system login</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="detachDatabases">
           <property name="toolTip">
            <string>Detach block and address databases at shutdown. This means they can be moved to another data directory, but it slows down shutdown. The wallet is always detached.</string>
           </property>
           <property name="text">
            <string>&amp;Detach databases at shutdown</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_Main">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabNetwork">
        <attribute name="title">
         <string>&amp;Network</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_Network">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="mapPortUpnp">
             <property name="toolTip">
              <string>Automatically open the ArtBoomerang client port on the router. This only works when your router supports UPnP and it is enabled.</string>
             </property>
             <property name="text">
              <string>Map port using &amp;UPnP</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="connectSocks">
             <property name="toolTip">
              <string>Connect to the ArtBoomerang network through a SOCKS proxy (e.g. when connecting through Tor).</string>
             </property>
             <property name="text">
              <string>&amp;Connect through SOCKS proxy:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="proxyWarning">
           <item>
            <widget class="QLabel" name="statusProxy">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayoutProxy"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_Network">
           <item>
            <widget class="QLabel" name="proxyIpLabel">
             <property name="text">
              <string>Proxy &amp;IP:</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="buddy">
              <cstring>proxyIp</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QValidatedLineEdit" name="proxyIp">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>IP address of the proxy (e.g. 127.0.0.1)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="proxyPortLabel">
             <property name="text">
              <string>&amp;Port:</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="buddy">
              <cstring>proxyPort</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="proxyPort">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Port of the proxy (e.g. 9050)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="socksVersionLabel">
             <property name="text">
              <string>SOCKS &amp;Version:</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="buddy">
              <cstring>socksVersion</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QValueComboBox" name="socksVersion">
             <property name="toolTip">
              <string>SOCKS version of the proxy (e.g. 5)</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_Network">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_Network">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabWindow">
        <attribute name="title">
         <string>&amp;Window</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="labelThemes">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Theme</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QCheckBox" name="minimizeOnClose">
           <property name="toolTip">
            <string>Minimize instead of exit the application when the window is closed. When this option is enabled, the application will be closed only after selecting Quit in the menu.</string>
           </property>
           <property name="text">
            <string>M&amp;inimize on close</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QValueComboBox" name="comboBoxThemes">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Dark Style</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Light Style</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QCheckBox" name="minimizeToTray">
           <property name="toolTip">
            <string>Show only a tray icon after minimizing the window.</string>
           </property>
           <property name="text">
            <string>&amp;Minimize to the tray instead of the taskbar</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <spacer name="verticalSpacer_Window">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabDisplay">
        <attribute name="title">
         <string>&amp;Display</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_Display">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_1_Display">
           <item>
            <widget class="QLabel" name="langLabel">
             <property name="text">
              <string>User Interface &amp;language:</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="buddy">
              <cstring>lang</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QValueComboBox" name="lang">
             <property name="toolTip">
              <string>The user interface language can be set here. This setting will take effect after restarting ArtBoomerang.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2_Display">
           <item>
            <widget class="QLabel" name="unitLabel">
             <property name="text">
              <string>&amp;Unit to show amounts in:</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="buddy">
              <cstring>unit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QValueComboBox" name="unit">
             <property name="toolTip">
              <string>Choose the default subdivision unit to show in the interface and when sending coins.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="displayAddresses">
             <property name="toolTip">
              <string>Whether to show ArtBoomerang addresses in the transaction list or not.</string>
             </property>
             <property name="text">
              <string>&amp;Display addresses in transaction list</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="coinControlFeatures">
             <property name="toolTip">
              <string>Whether to show coin control features or not.</string>
             </property>
             <property name="text">
              <string>Display coin &amp;control features (experts only!)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacerDisplay">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabAccounts">
        <attribute name="title">
         <string>External &amp;Accounts</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string> External servers settings</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="eAccLabelHelp">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollArea" name="eAccScrollArea">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustIgnored</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="eAccScrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>518</width>
              <height>288</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="eAccEmailLayout">
               <item>
                <layout class="QFormLayout" name="eAccForm">
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="eAccSmtp">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QValueComboBox" name="eAccEmail"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>SMTP server</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="eAccSmtpPort">
                   <property name="inputMethodHints">
                    <set>Qt::ImhDigitsOnly</set>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>SMTP Port</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Email</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QFormLayout" name="eAccForm_2">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetNoConstraint</enum>
                 </property>
                 <property name="rowWrapPolicy">
                  <enum>QFormLayout::DontWrapRows</enum>
                 </property>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="eAccUname"/>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="eAccUnameLabel">
                   <property name="text">
                    <string>Username</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="eAccPwd">
                   <property name="inputMethodHints">
                    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="eAccPwdLabel">
                   <property name="text">
                    <string>Password</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="eAccMailFrom">
                   <property name="inputMethodHints">
                    <set>Qt::ImhEmailCharactersOnly</set>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="eAccMailFromLabel">
                   <property name="text">
                    <string>Mail From</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
       <item>
        <spacer name="horizontalSpacer_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>48</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>&amp;OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="applyButton">
         <property name="text">
          <string>&amp;Apply</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>bitcoinamountfield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qvalidatedlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QValueComboBox</class>
   <extends>QComboBox</extends>
   <header>qvaluecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
